lib/cloudinary/utils.rb

```diff
@@ -1132,4 +1133,15 @@ def self.patch_fetch_format(options={})
   def self.is_remote?(url)
     REMOTE_URL_REGEX === url
   end
+
+  def self.compute_signature(message, long_url_signature)
+    encode_digest =
+      if long_url_signature
+        Base64.urlsafe_encode64(Digest::SHA256.digest(message))[0, 32]
```

Comment:
Please refactor this code, remove duplicate function calls, for example, `Base64.urlsafe_encode64` can be called once, after appropriate digest is calculated.